verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx

```diff
@@ -96,7 +97,7 @@ describe('<LandingPage />', () => {
 });
 
 describe('<SignInForm />', () => {
-    it('renders and submits form', async () => {
+    it.skip('renders and submits form', async () => {
```

Comment:
Are these temporary skips? Will be good to create an issue to keep track if it needs to be un-skipped

Reply:
Yes, I indended to create an issue fot "unskipping" the tests, but I must have forgotten. Here it is:
https://github.com/globaldothealth/turnkey-curator-portal/issues/129